docs/release/role-handbooks/release-lead/README.md

```diff
@@ -168,6 +168,11 @@ The goal of this task is to ensure we are always using the latest Go version for
 
 Note: If the Go minor version of one of our supported branches goes out of supported, we should consider bumping
```

Comment:
```suggestion
Note: If the Go minor version of one of our supported branches goes out of support, we should consider bumping
```
If you don't mind, could you kindly fix this nit

Reply:
Sure, no problem